addons/website_event/views/event_templates_list.xml

```diff
@@ -12,6 +12,12 @@
             <!-- Options -->
             <t t-set="opt_events_list_cards" t-value="is_view_active('website_event.opt_events_list_cards')"/>
             <t t-set="opt_events_list_columns" t-value="is_view_active('website_event.opt_events_list_columns')"/>
+            <t t-set="opt_events_list_rows" t-value="is_view_active('website_event.opt_events_list_rows')"/>
+            <t t-set="opt_index_sidebar" t-value="is_view_active('website_event.opt_index_sidebar')"/>
+            <!-- Qweb variable defining the class suffix for navbar items.
+                 Change accordingly to the derired visual result (eg. `primary`, `dark`...)-->
+            <t t-set="navClass" t-valuef="light"/>
```

Comment:
Not sure I understand this. We use it only once inline, could we not get rid of it?